tools/pot/docs/Introduction.md

```diff
@@ -16,6 +16,8 @@
 
 
 
+.. note:: Post-training Optimization Tool is deprecated since OpenVINO 2023.0. **Neural Network Compression Framework (NNCF)** is recommended for the post-training quantization instead.
```

Comment:
we might need to add the same note in more POT doc places just in case, like POT FAQ, and probably link to migration tutorial in POT API doc, need to take a look

Reply:
I don't really want to add it to every document but feel free to suggest ones where it could be useful. Added note to POT FAQ.